packages/@vuepress/theme-default/components/DropdownLink.vue

```diff
@@ -33,14 +35,17 @@
             <li
               class="dropdown-subitem"
               :key="childSubItem.link"
-              v-for="childSubItem in subItem.items"
+              v-for="(childSubItem, subIndex) in subItem.items"
             >
-              <NavLink :item="childSubItem"/>
+              <NavLink
+                @keypressTab="focusoutDropdown((index + 2) * (subIndex + 1))"
```

Comment:
I don't understand the `+ 2`? Could you explain? Why not just `+ 1`?

Reply:
I'm sorry, There was a bug here. I resolved this. 👍